app/models/prediction.rb

```
@@ -83,8 +91,67 @@ def predicted_score_string
     end
   end
 
+  ##
+  # Returns a formatted string showing the predicted scores for both teams with their names, adjusting the display to avoid ties by incrementing one team's score if the rounded scores are equal but the raw scores differ.
```

RuboCop (1.75.5)

[convention] Line is too long. [220/140]

(Layout/LineLength)